<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_uid">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>UID:{}的评分结果</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_avatars">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_icon">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_type">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>圣遗物类型：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_suit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>套装：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_stat">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>词条：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_result">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_icon_4">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_type_4">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>圣遗物类型：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_suit_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>套装：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_stat_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>词条：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_result_4">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_icon_2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_type_2">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>圣遗物类型：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_suit_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>套装：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_stat_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>词条：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_result_2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButton_icon_5">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_type_5">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>圣遗物类型：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_suit_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>套装：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_stat_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>词条：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_result_5">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_icon_3">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_type_3">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>圣遗物类型：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_suit_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>套装：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_stat_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>114514</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>词条：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_result_3">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
